distribution/tools/keystore-cli/src/test/java/org/opensearch/common/settings/AddFileKeyStoreCommandTests.java

```diff
@@ -85,14 +86,17 @@ public void testMissingCreateWithEmptyPasswordWhenPrompted() throws Exception {
     }
 
     public void testMissingCreateWithEmptyPasswordWithoutPromptIfForced() throws Exception {
+        assumeFalse("Can't use empty password in a FIPS JVM", inFipsJvm());
         String password = "";
         Path file1 = createRandomFile();
         execute("-f", "foo", file1.toString());
         assertSecureFile("foo", file1, password);
     }
 
     public void testMissingNoCreate() throws Exception {
-        terminal.addSecretInput(randomFrom("", "keystorepassword"));
+        var password = randomFrom("", "keystorepassword");
```

Comment:
Can we change the logic for tests like this to ensure they run in FIPS mode? Instead of the call to `randomFrom("", "keystorepassword")` does it make sense to always run this test with `keystorepassword` as the value for password if running in FIPS mode?

IMO I would rather run the tests then have them being skipped randomly if the pw chosen is empty.